public/scripts/app.js

```diff
@@ -26,17 +26,17 @@ $(document).ready(function(){
   // make a GET request for all restaurants
   $.ajax({
     method: 'GET',
-    url: '/api/restaurant',
+    url: '/api/restaurants',
```

Comment:
For API index endpoints, always use plural (`/api/restaurants`, not `/api/restaurant`).